mmyolo/deploy/models/dense_heads/yolov5_head.py

```diff
@@ -146,3 +146,34 @@ def yolov5_head__predict_by_feat(ctx,
 
     return nms_func(bboxes, scores, max_output_boxes_per_class, iou_threshold,
                     score_threshold, pre_top_k, keep_top_k)
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmyolo.models.dense_heads.yolov5_head.'
+    'YOLOv5Head.predict',
+    backend='rknn')
+def yolov5_head__predict__rknn(ctx, self, x: Tuple[Tensor], *args, **kwargs) -> Tensor:
+                               **kwargs) -> Tuple[Tensor, Tensor, Tensor]:
```

Comment:
maybe something wrong after the merge 😢

Reply:
Not familiar with `Allow edits by maintainers`, closed now.